Sources/xcbeautify/Xcbeautify.swift

```diff
@@ -23,6 +23,9 @@ struct Xcbeautify: ParsableCommand {
     @Flag(name: .long, help: "Disable the colored output")
     var disableColoredOutput = (ProcessInfo.processInfo.environment["NO_COLOR"] != nil)
 
+    @Option(help: "Specify an OutputFormat.")
```

Comment:
The Swift type `OutputFormat` is an implementation detail that's not known when interfacing with the CLI. I'd say something along the lines of:

```
Specify the output format (e.g. `--github-action`)
```